cli/plasmo/src/features/extension-devtools/generate-icons.ts

```diff
@@ -32,7 +32,7 @@ export async function generateIcons(
 
         return existsSync(developerProvidedImagePath)
           ? copy(developerProvidedImagePath, generatedIconPath)
-          : image512.resize({ width }).save(generatedIconPath)
+          : image512.resize({ width, height: width }).toFile(generatedIconPath)
```

Comment:
If we use the default resizing scheme it seems to be called "cover" which clips/crops the image to get a smaller image

[`Preserving aspect ratio, ensure the image covers both provided dimensions by cropping/clipping to fit`](https://sharp.pixelplumbing.com/api-resize#resize)

Do we want to specify `contain` here, instead?

Reply:
If you've tested it and it doesn't clip, maybe the docs are wrong but just wanna double check

Reply:
I think it only clips if the image is not squared. I tested it with square image (our provided icon) and it worked well!